src/openrct2/ride/Vehicle.cpp

```diff
@@ -9071,10 +9061,8 @@ static int32_t vehicle_update_track_motion_mini_golf(rct_vehicle* vehicle, int32
 
 loc_6DC9BC:
     _vehicleMotionTrackFlags |= VEHICLE_UPDATE_MOTION_TRACK_FLAG_5;
-    regs.eax = vehicle->remaining_distance + 1;
-    _vehicleVelocityF64E0C -= regs.eax;
-    vehicle->remaining_distance -= regs.eax;
-    regs.ebx = vehicle->vehicle_sprite_type;
```

Comment:
ebx is still used in 6dcd2b

Reply:
https://github.com/OpenRCT2/OpenRCT2/blob/57471edc47b9fbb8461a60f28c4ad59fb02a316c/src/openrct2/ride/Vehicle.cpp#L9216-L9223

Doesn't look like it is